elements/pfe-primary-detail/demo/pfe-primary-detail.story.js

```diff
@@ -1,14 +1,6 @@
-// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-// !! This is a placeholder
-// !!
-// !! Having a hard time understanding how to get this setup,
-// !! leaving it with a working sample that does not respond to
-// !! story book configuration.
-// !!
-// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-
 import { storiesOf } from "@storybook/polymer";
-import * as storybookBridge from "@storybook/addon-knobs";
+import * as bridge from "@storybook/addon-knobs";
```

Comment:
@wesruv Would love you review of this storybook update when you get a chance - I updated it when I was working on the codeblock injection